libraries/resource_container/ResourceContainer.py

```diff
@@ -240,7 +240,8 @@ def chapters(self, identifier=None):
             return []
         else:
             chapters = []
-            for d in sorted(glob(os.path.join(self.path, p.path, '*'))):
+            g = glob(os.path.join(self.dir, p.path, '*')
+            for d in sorted(glob(os.path.join(self.dir, p.path, '*')),key=lambda path: '{:0>6}'.format(os.path.basename(path))):
```

Comment:
Make sure there is a test case that verifies ordering here is done numerically. Have it preprocess a zip file from a PSA repo.

Reply:
Make test case with: https://git.door43.org/tx-manager-test-data/ceb_psa_text_ulb_L3

Reply:
Also, while pretty much the same thing, this is a bit more readable:

`key=lambda path: os.path.basename(path).zfill(3)`

(and we only really need to pad 3 0s as no book has # of books in the thousands).